ui/src/logs/components/LogsTable.tsx

```diff
@@ -236,6 +246,7 @@ class LogsTable extends Component<Props, State> {
                   }}
                   setScrollTop={this.handleScrollbarScroll}
                   scrollTop={this.state.scrollTop}
+                  scrollLeft={this.state.scrollLeft}
```

Comment:
This wasn't explicitly set here before but should be now since scrollLeft can differ from the scrollLeft from fancy scroll after a new search